### DISCUSSION TITLE
Spunk Connect for Strimzi?

### DISCUSSION THREAD

**Discussion by davehouser1** [strimzi/strimzi-kafka-operator#5099] [General]
Hello, is there support for splunk connect via the Strimzi operator? I see that Strimzi has a kafka connect pod support, not sure splunk connect is the same or not.

Details on splunk connect: [here](https://www.splunk.com/en_us/blog/tips-and-tricks/splunk-connect-for-kafka-connecting-apache-kafka-with-splunk.html)
Here is the github: [here]( https://github.com/splunk/kafka-connect-splunk)
Info on configuration: [here](https://docs.splunk.com/Documentation/KafkaConnect/2.0.2/User/ConfigureSplunkKafkaConnect)

**Comment by scholzj** [comment]
From the links it looks like a connector plugin for Kafka Connect. So you should be able to use it. You will need to add the plugin to the Strimzi container image (there are several mechanisms, for more info see https://strimzi.io/docs/operators/latest/full/deploying.html#using-kafka-connect-with-plug-ins-str). Adn then you can create the connector instance (again, several different ways: https://strimzi.io/docs/operators/latest/full/deploying.html#con-creating-managing-connectors-str). I never used this particular connector, maybe someone else who used it will chime in.

**Reply by davehouser1** [reply]
Great! To confirm, I will need to first deploy a Strimzi Kafka Connect pod correct?
I am new to Kafka Connect and its integrations, so sorry if these questions are self explanatory.

**Reply by scholzj** [reply]
Yeah, you need to first deploy Kafka Connect with this connector plugin (to have the Splunk connector JARs available there) and then you create the connector.
